examples/ensembles.ipynb

```diff
@@ -31,7 +31,7 @@
   },
```

Comment:
<p>In addition, you can <code>weights</code> with the literal <code>auto</code>. In this case, the weights of pipelines are assigned with the importances got from <code>feature_importance_</code> property of <code>regressor</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/990/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>